### PR TITLE
feat: add no results case & set chartGroupBy when select new groupby

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
please see order by commits

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/CostAnalysisPage.vue [hunk 1]
```diff
@@ -39,12 +39,12 @@ const costAnalysisPageStore = useCostAnalysisPageStore();
...
 
 /* util */
-const setQueryOptions = (options?: CostQuerySetOption) => {
+const setQueryOptions = (options?: CostQuerySetModel['options']) => {
     if (options) costAnalysisPageStore.setQueryOptions(options);
     else costAnalysisPageStore.initState();
 };
 
-const getQueryOptionsFromUrlQuery = (urlQuery: CostAnalysisPageUrlQuery): CostQuerySetOption => ({
+const getQueryOptionsFromUrlQuery = (urlQuery: CostAnalysisPageUrlQuery): CostQuerySetModel['options'] => ({
```
Comment (piggggggggy) [inline apps/web/src/services/cost-explorer/cost-analysis/CostAnalysisPage.vue:47]: 🤭

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue [hunk 2]
```diff
@@ -77,6 +83,7 @@ const handleChangeDefaultGroupBy = async (selectedItems: GroupBySelectButtonItem
         const addedGroupByName: string = xor(costAnalysisPageState.groupBy, selectedItems.map((d) => d.name))[0];
         costAnalysisPageStore.$patch((_state) => {
             _state.groupBy = [addedGroupByName, ..._state.groupBy];
+            _state.chartGroupBy = addedGroupByName;
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue:86]: question: what is chartGroupBy?

**Review by WANZARGEN — APPROVED** [review]
Well done on the content. I have a question, though.

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue:86]: 스크린샷 2023-09-11 오전 11 08 28

This! chartGroupBy is only one.

**Pull request merged**